src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -1,16 +1,54 @@
-import React, { useMemo, useState, useEffect } from "react";
-import { useForm } from "react-hook-form";
+import React, { useCallback, useMemo, useState } from "react";
+import { Dropdown, DropdownButton } from "react-bootstrap";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faPaperPlane } from "@fortawesome/free-solid-svg-icons";
+import {
+  faPoll,
+  faQuestion,
+  IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
 
 import { MessageToDisplay } from "types/chat";
 
 import { WithId } from "utils/id";
 
 import { ChatMessage } from "components/atoms/ChatMessage";
+// import { ChatPoll } from "components/atoms/ChatPoll";
+import ChatMessageBox from "../ChatMessageBox";
+import PollBox from "../PollBox";
```

Comment:
Presumably these are being loaded from `component/molecules/`? If so, we use the full pathname to them rather than a relative (eg. `../`) path. Also, `molecules/*` imports get sorted before `atoms/*`.

```suggestion
import ChatMessageBox from "components/molecules/ChatMessageBox";
import PollBox from "components/molecules/PollBox";

import { ChatMessage } from "components/atoms/ChatMessage";
// import { ChatPoll } from "components/atoms/ChatPoll";
```